src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -26,6 +26,7 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Strings;
 import com.jfoenix.controls.JFXButton;
+import com.jfoenix.controls.JFXDialog;
```

Comment:
unused, try turning on the option when commiting that removes unused imports

Reply:
Done

Reply:
Not done

Reply:
The JFXDialog is used by uiService.showInDialog on line 406 which I thought we were going to leave in if not I can try to find a way to remove it.